tests/mock_network/mod.rs

```diff
@@ -467,19 +451,16 @@ fn simultaneous_joining_nodes_three_section_with_one_ready_to_split() {
         SimultaneousJoiningNode {
             dst_section_prefix: small_prefix,
             src_section_prefix: small_prefix,
-            dst_relocation_interval_prefix: Some(small_prefix_to_add),
```

Comment:
Does this test still does what it was suppose to do?

Reply:
I would say it does what it was doing before this PR, but that might not necessary be what we want. This is because the relocation_interval override wasn't actually used (only the relocation destination).

Reply:
Then maybe we broke the test before, because I think this test relied on the node going to the right sub section (like the other relocate during split maybe). Maybe could add an issue to fix it after (with the other test). Maybe the solution is that during test, when a joning node look for its new FullId, if mock_base it will look at some override and generate a full id in that prefix?

Reply:
Yeah, I think we should revise this test to make sure it's doing what we want it to do. But my point was that it wasn't this PR that changed it.

Reply:
Created the issue: https://github.com/maidsafe/routing/issues/1935